theano/sandbox/gpuarray/blas.py

```diff
@@ -239,6 +239,7 @@ def c_code(self, node, name, inputs, outputs, sub):
         dims[0] = PyGpuArray_DIMS(%(A)s)[0];
         dims[1] = PyGpuArray_DIMS(%(B)s)[1];
 
+        Py_XDECREF(%(out)s);
```

Comment:
In fact, it is so fast to implement it, can you do it instead of making a ticket?